dask/array/core.py

```diff
@@ -3838,6 +3838,21 @@ def repeat(a, repeats, axis=None):
     return concatenate(out, axis=axis)
 
 
+@wraps(np.tile)
+def tile(A, reps):
+    if not isinstance(reps, Integral):
+        raise NotImplementedError("Only integer valued `reps` supported.")
```

Comment:
As you like.  Out of curiosity how large is the tiling argument?  How deep of a recursion and how many function calls are we talking here?